QuizApp/QuizApp/Storage/SecureStorage.swift

```diff
@@ -22,4 +22,8 @@ class SecureStorage: SecureStorageProtocol {
         keychain.set(accessToken, forKey: accessTokenKey)
     }
 
+    func deleteAccessToken() {
```

Comment:
Where is this used? I am not saying it should not be used but I cannot find it in the code.